website_sale_custom_filter/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Camptocamp
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "eCommerce Product Filters",
+    "category": "Website/Website",
+    "sequence": 50,
+    "summary": "Advanced website product filters",
+    "website": "https://github.com/OCA/e-commerce",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "version": "15.0.0.1.0",
+    "development_status": "Alpha",
+    "depends": ["website", "sale", "mail", "website_sale", "product"],
```

Comment:
```suggestion
    "depends": ["website_sale"],
```
Other modules are implicit dependencies of website_sale, so no need to list them.